packages/replay/src/replay.ts

```diff
@@ -546,6 +552,19 @@ export class ReplayContainer implements ReplayContainerInterface {
     this._context.urls.push(url);
   }
 
+  /**
+   * This will get the parametrized route name of the current page.
+   * This is only available if performance is enabled, and if an instrumented router is used.
+   */
+  public getCurrentRoute(): string | undefined {
+    const lastTransaction = this.lastTransaction || getCurrentHub().getScope().getTransaction();
+    if (!lastTransaction || lastTransaction.metadata.source !== 'route') {
```

Comment:
we can technically make this check `route` || `custom`, since we label as source `custom` if someone has custom transaction names - and we consider those parameterized.

Reply:
ah, good to know, that makes sense! 👍